modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/groups.ftl

```diff
@@ -49,8 +54,12 @@
 	${csvFileWriter.write("repository", virtualHostModel.hostname + "," + groupModel.friendlyURL + "," + groupId + ", " + groupModel.name + "\n")}
 </#list>
 
-<#assign defaultSiteHomePageContentLayoutModels = dataFactory.newContentPageLayoutModels(guestGroupModel.groupId, "home") />
+<#assign
+	defaultSiteHomePageContentLayoutModels = dataFactory.newContentPageLayoutModels(guestGroupModel.groupId, "home")
+	defaultSiteHomePageSegmentsExperienceModel = dataFactory.newSegmentsExperienceModel(defaultSiteHomePageContentLayoutModels)
+/>
 
+${dataFactory.toInsertSQL(defaultSiteHomePageSegmentsExperienceModel)}
```

Comment:
@slnn Plz add a new line after this line.

Reply:
@slnn We can forward after finishing this.

Reply:
@dantewang Done